src/boot/efi/stub.c

```diff
@@ -253,29 +254,113 @@ static EFI_STATUS load_addons_from_dir(
 
 }
 
```

Comment:
btw, please add a new EFI_STUB_FEATURE_XYZ feature flag for this, and display it in bootctl.

i.e. EFI_STUB_FEATURE_DEVICETREE_ADDONS or so, mirroring the existing EFI_STUB_FEATURE_CMDLINE_ADDONS